src/lazy/binary/raw/v1_1/binary_buffer.rs

```diff
@@ -535,12 +541,14 @@ impl<'a> BinaryBuffer<'a> {
             return Ok((None, *self));
         }
 
+        // Check for delimited container end opcode (0xEF) before trying to read FlexSym
+        if let Some(0xEF) = self.peek_next_byte() {
+            let after = self.consume(1);
+            return Ok((None, after));
+        }
+
```

Comment:
This should be removed. The name position is interpreted as a FlexSym and `0xEF` is a valid start for a 4byte flexsym value.